bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/handler/YamahaZoneThingHandler.java

```diff
@@ -428,15 +426,15 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                                 inputWithPlayControl.previousTrack();
                                 break;
                         }
-                    } else if (command instanceof DecimalType) {
-                        int v = ((DecimalType) command).intValue();
+                    } else if (command instanceof DecimalType decimalType) {
+                        int v = decimalType.intValue();
                         if (v < 0) {
                             inputWithPlayControl.skipREV();
                         } else if (v > 0) {
                             inputWithPlayControl.skipFF();
                         }
-                    } else if (command instanceof StringType) {
-                        String v = ((StringType) command).toFullString();
+                    } else if (command instanceof StringType stringType) {
```

Comment:
Here also, can you consistently use either `stringType`, `commandAsString` or `stringCommand` in all lines?

Reply:
Done.